src/electron/create-menu.ts

```diff
@@ -384,6 +384,22 @@ export function createMenu(ctx: MenuContext): void {
 				{
 					type: 'separator'
 				},
+				{
+					label: 'Previous Page',
+					accelerator: 'CmdOrCtrl+Alt+Left',
+					enabled: ctx.store.getFirstPage() !== ctx.store.getCurrentPage(),
```

Comment:
This will evaluate to true if both `currentPage` and `firstPage` are undefined, e.g. when there are no pages at all. 
A more robust approach could be to try and get the previous page and check if it is not undefined, e.g.:

```
enabled: typeof ctx.store.getPreviousPage() !== 'undefined'
```